vizro-mcp/README.md

```diff
@@ -0,0 +1,193 @@
+
+## Available Tools (if client allows)
+
+The Vizro MCP server provides the following tools. In general you should not need to use them directly, but in special cases you could ask the LLM to call them directly to help it find its way.
+
+- `get_vizro_chart_or_dashboard_plan` - Get a structured step-by-step plan for creating either a chart or dashboard. Provides guidance on the entire creation process.
+- `get_model_JSON_schema` - Retrieves the complete JSON schema for any specified Vizro model, useful for understanding required and optional parameters.
+- `validate_model_config` - Tests Vizro model configurations by attempting to instantiate them. Returns Python code and visualization links for valid configurations.
+- `load_and_analyze_csv` - Loads a CSV file from a local path or URL into a pandas DataFrame and provides detailed analysis of its structure and content.
+- `validate_chart_code` - Validates the code created for a chart and returns feedback on its correctness.
+- `get_sample_data_info` - Provides information about sample datasets that can be used for testing and development.
+
+## Available Prompts (if client allows)
+
+- `create_starter_dashboard` - Use this prompt template to get started with Vizro dashboards.
+- `create_EDA_dashboard` - Use this prompt template to create an Exploratory Data Analysis (EDA) dashboard based on a local or remote CSV dataset
+- `create_vizro_chart` - Use this prompt template to create a Vizro styled plotly chart based on a local or remote CSV dataset
+
+## Sample data
+
+You can find a set of sample CSVs to try out in the [Plotly repository](https://github.com/plotly/datasets/tree/master).
+
+## Development or running from source
+
+If you are a developer, or if you are running Vizro-MCP from source, you need to clone the Vizro repo. To configure the Vizro MCP server details:
+
+**For Claude**: Add the following to your `claude_desktop_config.json` [found via Developer Settings](https://modelcontextprotocol.io/quickstart/user#2-add-the-filesystem-mcp-server):
+
+**For Cursor**: Add the following to `mcp.json` [found via the Cursor Settings](https://docs.cursor.com/context/model-context-protocol#configuration-locations):
+
+```json
+{
+  "mcpServers": {
+    "vizro-mcp": {
+      "command": "uv",
+      "args": [
+        "run",
+        "--directory",
+        "<PATH TO VIZRO>/vizro-mcp/",
+        "vizro-mcp"
+      ]
+    }
+  }
+}
+```
+
+Replace `<PATH TO VIZRO>` with the actual path to your Vizro repository. You may also need to provide the full path to your `uv` executable, so instead of `"uv"` you would use something like `"/Users/<your-username>/.local/bin/uv"`. To discover the path of `uv` on your machine, in your terminal app, type `which uv`.
+
+## Other cool MCP Servers
```

Comment:
This is nice but not relevant really? I'd remove it.

Reply:
I did this in a separate commit (https://github.com/mckinsey/vizro/pull/1151/commits/4fca688f01d4fbf638539def663593879fe27300) in case you want to revert this but leave the other changes.